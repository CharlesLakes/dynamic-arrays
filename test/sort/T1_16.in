10
7 3 36 65 41 33 23 83 30 23