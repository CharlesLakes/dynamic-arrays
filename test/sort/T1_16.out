3 7 23 23 30 33 36 41 65 83